hypothesis-python/src/hypothesis/stateful.py

```diff
@@ -682,6 +692,11 @@ def __repr__(self):
     def upcoming_name(self):
         return u"v%d" % (self.name_counter,)
 
+    def last_names(self, n):
+        assert self.name_counter > n
```

Comment:
I'm not sure what Hypothesis' opinion is in using `assert` in internal functions

Reply:
We are huge fans 😄